dashboard/index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Enduro</title>
   </head>
-  <body>
+  <body class="p-0">
```

Comment:
Somehow, showing the dialog adds some padding to the right of the body, this prevents that from happening.

Reply:
CSS is awesome :stuck_out_tongue_closed_eyes: 
![css-is-awesome](https://github.com/user-attachments/assets/ccdf4b5d-eebf-476f-9f41-9e7242c56547)